libs/dh/shared/feature-authorization/src/lib/permission.service.ts

```diff
@@ -66,7 +79,7 @@ export class PermissionService {
     const claims = window.atob(jwtPayload);
 
     const jwtJson = JSON.parse(claims);
-    const roles = jwtJson['roles'] as Permission[];
+    const roles = jwtJson['role'] as Permission[];
```

Comment:
question: when is the `role` property added to a token? Is it when a token is acquired via our BFF?

Reply:
Jep, the new token with role claim is acquired from markpart through bff